### PR TITLE
Larger css

### PR DESCRIPTION
**Pull request by benjamin-firth**
What's this PR do?
This PR adds an scss file for visual changes needed on screens larger than 1500px.  Spacing and widget size was altered, as well as header sizing.

Where should the reviewer start?
largerCss.scss file

How should this be manually tested?
N/A

Any background context you want to provide?
As we alter styling on the main page, so of this is subject to change.  This just sets a template for the current CSS.  Will need more research to see how to actually use this file as the styling on a media query in the main file.

What are the relevant tickets?
n/a
Screenshots (if appropriate)
n/a
Questions:
Is there a blog post? - no
Does the knowledge base need an update? - no
Does this add new dependencies which need to be added? -no

@brittanystoroz @khalidwilliams

### CODE REVIEW

File: src/css/largeCSS.scss [hunk 1]
```diff
@@ -0,0 +1,52 @@
+// Media Query for screen size over 1500px;
```
Comment (sertmer) [inline src/css/largeCSS.scss]: I'm looking forward to working with sass a little more - interesting to see the separate files... Seems like a much better way to organize it all. Definitely an improvement over the super long css files I'm used to.

File: src/ActivityRepository.js [hunk 2]
```diff
@@ -103,25 +103,16 @@ class ActivityRepository {
...
-    return this.user.reduce((acc, day, index) => {
-      if (index < 2) {
-        return acc;
-      }
-      if ((day.numSteps < this.user[index - 1].numSteps) &&
+      } 
+      if (!positive && (day.numSteps < this.user[index - 1].numSteps) &&
         (this.user[index - 1].numSteps < this.user[index - 2].numSteps)) {
         acc.push(day.date);
       }
```
Comment (sertmer) [inline src/ActivityRepository.js]: I'm a little confused to see my activity repo changes here - was your master not up to date?

File: src/css/largeCSS.scss [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
```
Comment (VeeAndrade) [inline src/css/largeCSS.scss]: Looking forward to seeing this on the big screen!

**Comment by brittanystoroz** [comment]
Yes just be careful of incorporating those unrelated changes into a PR that is supposed to be about CSS/Sass.

**Pull request merged**